src/dbus_names.h

```diff
@@ -36,14 +36,6 @@ namespace DBus {
         enum Type {
             BusName, UniqueName, WellKnownName, ErrorName,
             InterfaceName, NamespaceName, MemberName };
-        static constexpr std::array<const char*, 7> nameStr = {
```

Comment:
@mikecrowe this symbol was missing according to clang, has this to do with -fvisibility as well? If so what's happening?
Moving the symbol to the .cpp made it link again

Reply:
This might be similar to 7ca1104c5a49f0f30ac5c7e1bfdc7756d6dd19d3.

I must admit that I'm not at all expert with `-fvisibility`, especially with C++, and the need to use `-fuse-ld=gold` in ff563e1a3e003dbd904251e2e0c587ad5861a2cb doesn't inspire confidence that all is well. Does the problem go away if you revert ff563e1a3e003dbd904251e2e0c587ad5861a2cb?

Reply:
Reverting didn't help: https://github.com/d12fk/dbus-asio/runs/6042722273
I have no idea what clang doesn't like here.

Reply:
This works with -std=c++17, so I guess there's an issue with std::array that's resolved in later a std. For now I would like to keep it to -std=c++1y, as RHEL7 is that far back and supported for another year.

Reply:
> RHEL7 is that far back and supported for another year.

RHEL7 is supported until end of June 2024, so two more years.  It by default ships with GCC 4.8.5 (IIRC), which only supports -std=c++1y (which is most of, but not all, C++14 features).  That said, the `devtoolset-10` software library is available, which gives GCC-10.3, with should support `-std=c++17`.  OpenVPN 3 Linux is already being built with this compiler for the official releases.